files/private-chef-cookbooks/private-chef/libraries/private_chef.rb

```diff
@@ -37,6 +37,7 @@ module PrivateChef
   opscode_chef Mash.new
 
   lb Mash.new
+  lb['xdl_defaults'] ||= Mash.new
```

Comment:
Just curious why this had to be added now, given that other settings like `default['private_chef']['lb']['xdl_defaults']['couchdb_associations']` exist. Is it just because we're exposing a dark launch setting via the server config file for the first time?